src/pages/api/login.ts

```diff
@@ -0,0 +1,17 @@
+// ./pages/api/login
+import {NextApiRequest, NextApiResponse} from 'next'
+import {setAuthCookies} from 'next-firebase-auth'
+import initAuth from 'utils/firebase/initAuth' // the module you created above
+
+initAuth()
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    await setAuthCookies(req, res)
+  } catch (e) {
+    return res.status(500).json({error: 'Unexpected error.'})
+  }
+  return res.status(200).json({success: true})
+}
+
+export default handler
```

Comment:
This route is used by `next-firebase-auth` to set the cookies it needs to tell if the user is logged in.